src/Movie-details/MovieDetails.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react'
+// import Ratings from '../Ratings/Ratings'
+// import Details from '../Details/Details'
+import { getMovieDetails } from '../apiCalls'
+import '../scss/_MovieDetails.scss'
+
+class MovieDetails extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      movieId: this.props.movieId,
+      title: '',
+      poster: '',
+      backdrop: '',
+      releaseDate: '',
+      overview: '',
+      averageRating: '',
+      genres: [],
+      budget: 'number',
+      revenue: 'number',
+      runtime: 'number',
+      tagline: '',
+      videos: [],
+      error: '',
+    }
+  }
+  
+  componentDidMount() {
+    getMovieDetails(this.state.movieId)
+      .then(data => this.setMovieInfo(data))
+      .catch(error => {
+        console.log(error)
+        this.setState({ error: error })
```

Comment:
We should add in an error message here that we can display for the user in case there is an issue getting the movie details.

Reply:
Some conditional rendering for that would be nice!

Reply:
Also make sure that you check to see whether the fetch call gives you back a 200-level response as part of your error handling. Check out the [ok](https://developer.mozilla.org/en-US/docs/Web/API/Response/ok) property of the response object.

Reply:
I like that you're making this request in the details component, though.